src/app/src/components/Sidebar.js

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Box } from 'rebass';
-import { Heading } from './custom-styled-components';
+import { Heading, Text } from './custom-styled-components';
 import { themeGet } from 'styled-system';
 
 const StyledSidebar = styled(Box)`
-    width: 300px;
-    background: ${themeGet('colors.lightgray')};
+    background: ${themeGet('colors.teals.5')};
+    text-align: center;
+    width: 100%;
 `;
 
-const Sidebar = () => {
+const Sidebar = ({ fish }) => {
+    const { commonName, scientificName, path, notes, timestamp } = fish;
+    const date = new Date(timestamp).toLocaleString();
+    const video = fish && <video src={`${path}.mp4`} autoPlay={true} />;
```

Comment:
We need to add calls to `pauseTimer()` and `resetTimer()` here:
 `onPlay={() => dispatch(pauseTimer())}`
```suggestion
    const video = fish && (<video
         src={`${path}.mp4`}
         autoPlay={true}
         onPlay={() => dispatch(pauseTimer())}
         onEnded={() => dispatch(resetTimer())}
    />);
```

Reply:
This is a great suggestion!! Good use of the spread operator too. Tackled in latest commit